.goreleaser.yaml

```
@@ -18,7 +18,10 @@ builds:
 archives:
   - format: binary
     name_template: >-
-      {{ .ProjectName }}_{{ .Os }}_
+      {{ .ProjectName }}_
+      {{- if eq .Os "darwin" }}Darwin
+      {{- else if eq .Os "linux" }}Linux
+      {{- else }}{{ .Os }}{{ end -}}_
```

it's not wrong, but it can be simplified to

```suggestion
      {{- title .Os }}_
```